### ISSUE TITLE
Proto Nets mini imagenet results are not matching reported results.

### ISSUE THREAD

**Issue by mayug** [learnables/learn2learn#219]
When I ran protonet_miniimagenet.py, the test accuracy is only around 43% instead of 49%. Why is this? Any idea?

**Comment by seba-1511** [comment]
Hello @mayug , could you specify which command you ran? It's probably a matter of hyper-parameter choice.

**Comment by mayug** [comment]
Hi, 
I ran `python protonet_miniimagenet.py --gpu 1`  for 1 shot 5 ways. 
All other arguments were defaults.
Thanks

**Comment by mayug** [comment]
These were the defaults.

```
    parser.add_argument('--max-epoch', type=int, default=250)
    parser.add_argument('--shot', type=int, default=1)
    parser.add_argument('--test-way', type=int, default=5)
    parser.add_argument('--test-shot', type=int, default=1)
    parser.add_argument('--test-query', type=int, default=30)
    parser.add_argument('--train-query', type=int, default=15)
    parser.add_argument('--train-way', type=int, default=30)

    optimizer = torch.optim.Adam(model.parameters(), lr=0.001)
    lr_scheduler = torch.optim.lr_scheduler.StepLR(
        optimizer, step_size=20, gamma=0.5)
```

**Comment by mayug** [comment]
@seba I think I figured it out.
https://github.com/learnables/learn2learn/blob/9fe3a505da6a09640dadd101bb679df57726c030/examples/vision/protonet_miniimagenet.py#L32

Should include maxpool=True in ConvBase.

Making this change gives 49% on Mini Imagenet.

Thank you for this project. 👍

**Comment by seba-1511** [comment]
Great, glad that you solved it! Do you want to send a PR for that?

**Comment by mayug** [comment]
@seba-1511 Sure. I have sent a PR for that.
